i6_models/parts/conformer/frontend.py

```diff
@@ -0,0 +1,282 @@
+from __future__ import annotations
+
+__all__ = [
+    "ConformerVGGFrontendV1Config",
+    "ConformerVGGFrontendV1",
+    "ConformerVGGFrontendV2Config",
+    "ConformerVGGFrontendV2",
+]
+
+from dataclasses import dataclass
+from typing import Callable, Optional, Tuple, Union
+
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass()
+class ConformerVGGFrontendV1Config(ModelConfiguration):
+    """
+    Attributes:
+        features: number of features for the initial conv layer
+        channels_conv_1_2: number of channels for conv layers
+        conv_kernel_size: kernel size of conv layers
+        pool_1_kernel_size: kernel size of first pooling layer
+        pool_1_strides: strides of first pooling layer
+        pool_2_kernel_size: kernel size of second pooling layer
+        pool_2_strides: strides of second pooling layer
+        activation: activation function applied after norm
+    """
+
+    features: int
+    channels_conv_1_2: int
+    channels_conv_3_4: int
+    conv_kernel_size: Union[int, Tuple[int, ...]]
+    pool_1_kernel_size: Union[int, Tuple[int, ...]]
+    pool_1_strides: Optional[Union[int, Tuple[int, ...]]]
+    pool_2_kernel_size: Union[int, Tuple[int, ...]]
+    pool_2_strides: Optional[Union[int, Tuple[int, ...]]]
+    activation: Union[nn.Module, Callable[[torch.Tensor], torch.Tensor]]
+
+    def check_valid(self):
+        if isinstance(self.conv_kernel_size, int):
+            assert self.conv_kernel_size % 2 == 1, "ConformerVGGFrontendV1 only supports odd kernel sizes"
+        if isinstance(self.pool_1_kernel_size, int):
+            assert self.pool_1_kernel_size % 2 == 1, "ConformerVGGFrontendV1 only supports odd kernel sizes"
+        if isinstance(self.pool_2_kernel_size, int):
+            assert self.pool_2_kernel_size % 2 == 1, "ConformerVGGFrontendV1 only supports odd kernel sizes"
+
+    def __post__init__(self):
+        super().__post_init__()
+        self.check_valid()
+
+
+class ConformerVGGFrontendV1(nn.Module):
+    """
+    Convolutional Front-End
+
+    Uses explicit padding for ONNX exportability, see:
+    https://github.com/pytorch/pytorch/issues/68880
+    """
+
+    def __init__(self, model_cfg: ConformerVGGFrontendV1Config):
+        """
+        :param model_cfg: model configuration for this module
+        """
+        super().__init__()
+
+        model_cfg.check_valid()
+
+        conv_padding = self._get_padding(model_cfg.conv_kernel_size)
+        pool_1_padding = self._get_padding(model_cfg.pool_1_kernel_size)
+        pool_2_padding = self._get_padding(model_cfg.pool_2_kernel_size)
+
+        self.conv1 = nn.Conv2d(
+            in_channels=model_cfg.features,
```

Comment:
I think this is wrong. See my comment below. I think the VGG operates on time and feature as the spatial dims. And you would add a dummy channel dim. So it means here:
```suggestion
            in_channels=1,
```